src/solver/hydro/management/daily.cpp

```diff
@@ -227,7 +227,7 @@ inline void HydroManagement::prepareDailyOptimalGenerations(
 {
     const auto srcinflows = area.hydro.series->storage.getColumn(y);
 
-    auto& data = tmpDataByArea_[&area];
+    auto& data = area.hydro.hydro_management_data[y];
```

Comment:
This is actually a pretty bad idea, since `area` is shared by threads you'll get race conditions, etc.

Reply:
ok